index.bs

```diff
@@ -4459,6 +4459,8 @@ In order to perform a [=registration ceremony=], the [=[RP]=] MUST proceed as fo
         - Otherwise, use the X.509 certificates returned as the [=attestation trust path=] from the [=verification procedure=]
             to verify that the attestation public key correctly chains up to an acceptable root certificate.
 
+            Note: The attestation certificate and the root certificate obtained above in step 20 can be the same certificate.
```

Comment:
I think maybe this note is not so helpful at this point. We should just modify line 4460 to change how we discuss validating the certificate. See other comment.

```suggestion
```

Reply:
@jcjones - While I support your suggestion for L4460 (thx!), I am unsure whether it is sufficiently clear to folks who are not as steeped in PKI-ness as you are. 

Perhaps we further update L4460 like so:
> ...verify that the attestation certificate either correctly chains up to an acceptable root certificate, or is itself an acceptable certificate (i.e., it and the root certificate obtained in step 20 may be the same).

..?

Reply:
OK! Works for me.